policies/testing.md

```diff
@@ -0,0 +1,42 @@
+Testing Policy
+==============
+
+This applies to all stable and development branches of the main code repository.
+Except where noted below:
+
+- All Pull Requests adding new functionality to the applications, libraries,
+providers or engines must include suitable tests.
+- All Pull Requests fixing a functional behavior defect in the applications,
+libraries, providers or engines must include a test for that defect.
```

Comment:
Do have or need a definition functional behaviour? Is any bugfix a functional behaviour change?

Reply:
I've added a paragraph attempting to explain what is meant by this.

Reply:
So any change that you can observe as caller is a function behaviour change.